src/java.base/share/classes/java/io/LineNumberReader.java

```diff
@@ -267,18 +267,7 @@ public String readLine() throws IOException {
     private char skipBuffer[] = null;
 
     /**
-     * Skip characters.
-     *
-     * @param  n
-     *         The number of characters to skip
-     *
-     * @return  The number of characters actually skipped
-     *
-     * @throws  IOException
-     *          If an I/O error occurs
-     *
-     * @throws  IllegalArgumentException
-     *          If {@code n} is negative
+     * {@inheritDoc}
```

Comment:
This appears to make the skip method disappear from the LineNumberReader javadoc
and only show up as a method declared in Reader.

Reply:
Right. This is intended and expected.